ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+Welcome to the issues section if it's your first time!
+
+### Before creating an issue, please ensure:
+- [ ] This is a bug in the software that resides in this repository, and not a
+      support matter (use https://otland.net/forums/support.16/ for support)
+- [ ] You have not made changes to the source code related to the problem
+- [ ] Search for a similar issue, add your reaction there instead :tada:
```

Comment:
This depends on the situation, and I prefer to not have it in the template. Sometimes the person who would open a duplicate issue may complement the original issue with additional information, and I'd rather read their comment than just see their reaction even though the issue will be closed and marked as duplicate.